db/index.js

```diff
@@ -1,7 +1,15 @@
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/listing_images');
+mongoose.connect('mongodb://localhost/listing_images', { useUnifiedTopology: true, useNewUrlParser: true });
```

Comment:
As far as I'm aware, there's an 80-character per line limit on the style guide. To avert this, I would recommend displaying the settings object across multiple lines, or by assigning the settings to a variable.